packages/backend/src/peripherals/database/ReportRepository.ts

```diff
@@ -14,13 +14,15 @@ export interface ReportRecord {
   balance: bigint
 }
 
+export const DAY = 86400
```

Comment:
I think I would prefer it as `UnixTimestamp.DAY` and `UnixTimestamp.SIX_HOURS` or even `UnixTimestamp.HOUR * 6`